src/modem/mod.rs

```diff
@@ -308,6 +340,53 @@ impl<'c, P: ModemPower> Modem<'c, P> {
         Ok(())
     }
 
+    pub fn reset_network_priority(&mut self) {
```

Comment:
This needs documentation. Really, the whole network priority system needs to be documented, otherwise users will be confused by it